db/migrate/20201030204952_add_uid_to_user.rb

```diff
@@ -1,5 +1,6 @@
 class AddUidToUser < ActiveRecord::Migration[5.2]
   def change
     add_column :users, :uid, :string
+    add_index :users, :uid
```

Comment:
Does this need to be added in a separate migration file? Or is it okay to just add to an existing migration?

Reply:
In the examples I saw online, it was included in the same migration. I thought that the schema looked fine.

Reply:
I think since it's merging schema too it should be good to go.